src/components/Header/Header.jsx

```
@@ -24,17 +24,17 @@ import { currentUserPropType } from '../../App/mermaidData/mermaidDataProptypes'
+import { useExploreLaunchFeature } from '../../library/useExploreLaunchFeature'
 import { UserIcon } from '../UserIcon/UserIcon'
 import BellNotificationDropDown from '../BellNotificationDropDown/BellNotificationDropDown'
 import HideShow from '../generic/HideShow'
 import Logo from '../../assets/mermaid-logo.svg'
 import OfflineHide from '../generic/OfflineHide'
 import ProfileModal from '../ProfileModal'
 
-const GlobalLinks = () => {
-  const { isAppOnline } = useOnlineStatus()
-  const mermaidDashboardLink = import.meta.env.VITE_MERMAID_DASHBOARD_LINK
+const GlobalLinks = ({ isAppOnline, isExploreLaunchEnabledForUser, mermaidExploreLink }) => {
```

ESLint

[error] 'isAppOnline' is missing in props validation

(react/prop-types)